app/services/invoices/issuing_date_service.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+module Invoices
+  class IssuingDateService < BaseService
```

Comment:
I don't think this should inherit from `BaseService` as it does not define the `call` method:

```suggestion
  class IssuingDateService
```